ofdb-core/src/usecases/send_update_reminders.rs

```diff
@@ -246,21 +246,43 @@ pub struct Reminder {
 }
 
 pub trait EmailReminderFormatter {
-    fn format_email(&self, reminder: &Reminder) -> EmailContent;
+    fn format_email(&self, reminder: &Reminder, review_nonce: &ReviewNonce) -> EmailContent;
 }
 
-pub fn create_emails<F>(
+pub fn create_reminder_review_tokens(reminders: Vec<Reminder>) -> Vec<(Reminder, ReviewToken)> {
+    reminders
+        .into_iter()
+        .map(|reminder| {
+            let expires_at = Timestamp::now() + time::Duration::days(30);
```

Comment:
Same here. Please try to get rid of the scattered `Timestamp::now()` invocations.